sympy/vector/tests/test_field_functions.py

```diff
@@ -11,70 +11,73 @@
                                     scalar_potential_difference)
 from sympy.utilities.pytest import raises
 
-C = CoordSysCartesian('C')
-i, j, k = C.base_vectors()
-x, y, z = C.base_scalars()
-delop = C.delop
+cartesian = CoordSystem3D('C')
+spherical = CoordSystem3D('spherical', DifferentialClass.build_instance_from_lambda(
```

Comment:
I like what this line is going, and the way it provides flexibility. But for users who want to initialize systems quick and efficiently, maybe we should build API (atleast for commonly used systems like spherical) that can do this with less effort?